reana_commons/mail.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA-Commons email util."""
+
+import logging
+import os
+import smtplib
+import ssl
+
+# Email configuration
+REANA_SMTP_SERVER = os.getenv('REANA_SMTP_SERVER')
+REANA_SMTP_PORT = os.getenv('REANA_SMTP_PORT')
+REANA_EMAIL_LOGIN = os.getenv('REANA_EMAIL_LOGIN')
+REANA_EMAIL_SENDER = os.getenv('REANA_EMAIL_SENDER')
+REANA_EMAIL_PASSWORD = os.getenv('REANA_EMAIL_PASSWORD')
+
+
+def send_email(receiver_email, subject, body, login_email=REANA_EMAIL_LOGIN,
+               sender_email=REANA_EMAIL_SENDER):
+    """Send emails from REANA platform."""
+    message = """\
+Subject: {subject}
+
+{body}""".format(subject=subject, body=body)
```

Comment:
Wouldn't this work as well? Seems a bit weird otherwise.
```suggestion
message = 'Subject: {subject}\n\n{body}'.format(subject=subject, body=body)
```